client/state/login/magic-login/actions.js

```diff
@@ -70,9 +81,58 @@ export const fetchMagicLoginRequestEmail = email => dispatch => {
 	} );
 };
 
+export const authError = error => {
+	return {
+		type: MAGIC_LOGIN_REQUEST_AUTH_ERROR,
+		error,
+	};
+};
+
+export const authSuccess = () => {
+	return {
+		type: MAGIC_LOGIN_REQUEST_AUTH_SUCCESS,
+	};
+};
```

Comment:
Let's make `authError` and `authSuccess` private, as we don't expect anything outside this module to use them.